src/components/pages/ArticlePage/index.module.css

```diff
@@ -0,0 +1,237 @@
+.main {
```

Comment:
I thought I might want to use Chakra UI's components here too, but it was a lot easier just to use microCMS's template. 

https://github.com/microcmsio/nextjs-simple-blog-template/blob/main/components/Article/index.module.css